addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/test/EmulatedV6Bridge.java

```diff
@@ -280,8 +280,8 @@ public void runBrigde() {
                                     + buffer[19]);
 
                             if (chksum != buffer[21]) {
-                                logger.error(
-                                        "Checksum wrong:" + String.valueOf(chksum) + " " + String.valueOf(buffer[21]));
+                                logger.error("Checksum wrong:{} {}", String.valueOf(chksum),
```

Comment:
_Look like some toStrings would be performed when needed but could require some testing._

I would expect `logger.error("Checksum wrong:{} {}", chksum, buffer[21]);` to work as well.